src/components/toast/global_toast_list.test.tsx

```diff
@@ -74,6 +74,36 @@ describe('EuiGlobalToastList', () => {
 
         expect(component).toMatchSnapshot();
       });
+      test('is side prop consistent', () => {
```

Comment:
This should be moved **outside** of the `toasts` describe block and update the name to:

```suggestion
      test('side can be changed to left', () => {
```

Reply:
I've made a new describe block for the side prop